Dockerfile

```diff
@@ -2,18 +2,20 @@ FROM golang:alpine
 
 RUN apk add --update git make sudo unzip wget openssh
 
+ADD . /go/src/github.com/HewlettPackard/terraform-provider-oneview
+
 RUN wget --no-check-certificate https://releases.hashicorp.com/terraform/0.6.16/terraform_0.6.16_linux_amd64.zip && \
     unzip terraform_0.6.16_linux_amd64.zip -d /usr/local/terraform && \
     rm terraform_0.6.16_linux_amd64.zip && \
     rm /usr/local/terraform/terraform-provider-* && \
     rm /usr/local/terraform/terraform-provisioner-chef
 
 RUN mkdir -p /go/src/github.com/HewlettPackard/ && \
-    cd /go/src/github.com/HewlettPackard && \
-    git clone https://github.com/HewlettPackard/terraform-provider-oneview.git && \
-    cd terraform-provider-oneview && \
-    git config --global http.sslVerify false && \
-    go get && \
+    cd /go/src/github.com/HewlettPackard/terraform-provider-oneview && \
+    wget --no-check-certificate https://glide.sh/get -O ./glide.sh && \
```

Comment:
this is how i do it to simply pin glide version:
```
ENV GLIDE_VERSION 0.12.1
ENV GLIDE_DOWNLOAD_URL https://github.com/Masterminds/glide/releases/download/v${GLIDE_VERSION}/glide-v${GLIDE_VERSION}-linux-amd64.tar.gz

RUN curl -fsSL "$GLIDE_DOWNLOAD_URL" -o glide.tar.gz \
    && tar -xzf glide.tar.gz \
    && mv linux-amd64/glide /usr/bin/ \
    && rm -r linux-amd64 \
    && rm glide.tar.gz
```

but it is not important, just to consider...